.github/workflows/make-json.yml

```diff
@@ -0,0 +1,33 @@
+name: Make JSON
+on:
+    pull_request:
+        branches: [main]
```

Comment:
Shouldn't this only build for things that have been merged?

```suggestion
```

Reply:
It should always build, but only deploy once it's merged. 
This is done with an `if` in the deploy step.

Reply:
Ah, cool, just saw that commit

I wrote the review last night, but only realized I hadn't posted it once I saw your ping 😅